libraries/tensorrt-6.0-cuda10.0/Dockerfile

```diff
@@ -0,0 +1,4 @@
+# This is a unique packageset in which it requires both a specific version of CUDA, and a version of Ubuntu as well.
+COPY --chown=0:0 tensorrt-6.0-cuda10.0/context/install /tmp/install
+RUN wget -O - https://james-s-public.s3.amazonaws.com/TensorRT-6.0.1.5.Ubuntu-16.04.x86_64-gnu.cuda-10.0.cudnn7.6.tar.gz | (cd /tmp && tar zxf -)
```

Comment:
I think we have a docker/wheels bucket specifically for pulling files. I would recommend putting it there instead of putting under `james-s-public`.

Reply:
What did you have in mind? Do we have any buckets that are accessable to us? (I no longer have our production aws account access)

Reply:
Yeah, we have a bucket called `algorithmia-docker` w/ a folder called `docker-deps` under it.

We should have read/write access assuming it hasn't been revoked – engineering has gone through a lot of changes over the last year, they may have accidentally revoked it.

Reply:
To be clear, the bucket is under the `prod` account.